### PR TITLE
Sputnik test

### PR DESCRIPTION
**Pull request by pjagielski**

### CODE REVIEW

File: src/main/java/pl/touk/throwing/ThrowingFunction.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.function.Function;
+import java.util.function.Function;
```
Comment (sputnikci) [inline src/main/java/pl/touk/throwing/ThrowingFunction.java]: [Checkstyle] ERROR: Duplicate import to line 20 - java.util.function.Function.

File: src/main/java/pl/touk/throwing/ThrowingFunction.java [hunk 2]
```diff
@@ -18,6 +18,7 @@
...
 import java.util.function.Supplier;
```
Comment (sputnikci) [inline src/main/java/pl/touk/throwing/ThrowingFunction.java]: [Checkstyle] ERROR: Redundant 'public' modifier.

File: src/main/java/pl/touk/throwing/ThrowingFunction.java [hunk 3]
```diff
@@ -71,7 +71,7 @@
         };
     }
```
Comment (pjagielski) [inline src/main/java/pl/touk/throwing/ThrowingFunction.java]: test

File: src/main/java/pl/touk/throwing/ThrowingFunction.java [hunk 4]
```diff
@@ -71,7 +71,7 @@
...
 
-    static <T, R, E extends Exception> Function<T, R> unchecked(ThrowingFunction<T, R, E> f) {
```
Comment (pjagielski) [inline src/main/java/pl/touk/throwing/ThrowingFunction.java]: test

File: src/main/java/pl/touk/throwing/ThrowingFunction.java [hunk 5]
```diff
@@ -71,7 +71,7 @@
...
+    public static <T, R, E extends Exception> Function<T, R> unchecked(ThrowingFunction<T, R, E> f) {
```
Comment (pjagielski) [inline src/main/java/pl/touk/throwing/ThrowingFunction.java]: test

**Pull request closed without merging**